TypeInjections/Tool/Tool.fs

```diff
@@ -21,31 +21,38 @@ module Tool =
         let commitRepo = newRepo.Lookup<Commit>(hash)
         Commands.Checkout(newRepo, commitRepo) |> ignore
 
-    // based off command line flags, create and provide path for input/output folders
-    let interpretFlag (argvList: string list) =
-        let flag = argvList.Item(0)
-
-        // -p [inputPath] [inputPath] [outputPath]
-        if flag.Contains("-p") then
-            Console.WriteLine("***** local project comparison")
-
-            if argvList.Length <> 4 then
-                failwith "usage: program -p OLD NEW OUTPUTFOLDER"
 
-        // -l [git commit hash] [git commit hash] [OutputPath]
-        elif flag.Contains("-l") then
-            Console.WriteLine("***** git commit")
+    type options =
+        { [<Option('d', "delete", Required = false, HelpText = "use option to remove New and Old folders")>]
+          delete: bool
+          [<Option('l', "local", Required = false, HelpText = "[inputPath] [inputPath] [outputPath]")>]
+          local: seq<string>
+          [<Option('g', "git", Required = false, HelpText = "[old commit] [new commit] [outputPath]")>]
+          git: seq<string>
+          [<Option('r', "remote", Required = false, HelpText = "[github repo url] [new commit] [old commit] [OutputPath]")>]
+          remote: seq<string>
+          [<Option(HelpText = "Prints all messages to standard output.")>]
+          verbose: bool }
 
-            if argvList.Length <> 4 then
-                failwith "usage: program -l OLD_COMMIT NEW_COMMIT OUTPUTFOLDER"
+    // based off command line flags, create and provide path for input/output folders
+    let interpretFlags  (local: string list) (git: string list) (remote: string list) : string * string * string =
+
+        if not local.IsEmpty then
+            Utils.log "***** local project comparison"
+            let oldFolderPath = local.Item(0)
+            let newFolderPath = local.Item(1)
+            let outFolderPath = local.Item(2)
+            oldFolderPath, newFolderPath, outFolderPath
+        else
 
+        if not git.IsEmpty then
+            Utils.log "***** git commit"
             // location of repo
             let location = Directory.GetCurrentDirectory()
 
-            let commit1 = argvList.Item(1)
-            let commit2 = argvList.Item(2)
-
-            let path = argvList.Item(3)
+            let commit1 = git.Item(0)
+            let commit2 = git.Item(1)
```

Comment:
inline `commit1` and `commit2`

Reply:
changed lines 53:
`let commit1, commit2 = git.Item(0), git.Item(1)`